source/mobile/mobile-troubleshoot.rst

```diff
@@ -43,7 +43,7 @@ As a workaround, you can bundle the ``js`` manually first with
 
 .. code-block:: none
 
-  react-native bundle --platform android --dev false --entry-file index.android.js --bundle-output  App/src/main/assets/index.android.bundle --assets-dest App/src/main/res/
+  --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res/
```

Comment:
I was only talking about the last arguments, sorry if that wasn't clear. The beginning of the command is still need.
So the complete command would be `react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res/`.

Reply:
All good :D Fixed it